ui/app/AppLayouts/Chat/popups/GroupChatPopup.qml

```diff
@@ -38,11 +38,11 @@ ModalPopup {
 
         contactList.membersData.append({
             //% "(You)"
-            name: popup.store.profileModelInst.profile.username + " " + qsTrId("(you)"),
-            pubKey: popup.store.profileModelInst.profile.pubKey,
+            name: userProfile.username + " " + qsTrId("(you)"),
+            pubKey: userProfile.pubKey,
```

Comment:
Shouldn't this still come from the store? Since we don't want views and popups to access the model directly.

Reply:
`userProfile` is a global instance, and it is available everywhere, on both nim and qml sides. But a concept is missed here, and this is just an adaption based on the code which already was there.

We're populating that `contactList` somehow artificially here, it should be maintained properly on the nim side, and just set as a model here, I hope that will be sorted out during refactor when it comes to the plate.